src/pages/how-to/monitor-system-and-network-activity.mdx

```diff
@@ -1,44 +1,35 @@
 
-# Monitor system and network activity
 
-The activity monitoring feature lets you quickly see what's happening with your network.
-Whether a new machine or user joined your network or the access control policy has been modified, the activity log allows you to track the changes to your network.
+# Monitor System and Network Activity
 
-## Access activity monitoring view
+The activity monitoring functionality in NetBird allows you to observe and track changes to your network infrastructure. This includes events such as when a new machine or user has joined your network, or when an access control policy has been modified.
```

Comment:
Should also add other events so that the user doesn’t have an impression that we only have two events